config/packages/framework.yaml

```diff
@@ -4,6 +4,8 @@ imports:
 # see https://symfony.com/doc/current/reference/configuration/framework.html
 framework:
   secret: '%secret%'
+  esi: false
```

Comment:
esi and fragments are both false by default. Do we still want the configure them explicitely?